src/panels/developer-tools/statistics/developer-tools-statistics.ts

```diff
@@ -172,22 +247,334 @@ class HaPanelDevStatistics extends SubscribeMixin(LitElement) {
   );
 
   protected render() {
+    const localize = this.hass.localize;
+    const columns = this._columns(this.hass.localize);
+
+    const selectModeBtn = !this._selectMode
```

Comment:
We should not duplicate all the logic from `hass-tabs-subpage-data-table` we should either use that element if possible and other wise extract it to another element so it can be reused in multiple elements.

Reply:
Any thought which way I should try to go? I had the same thought but couldn't really figure out how to make it work.

Reply:
I would say a new element that can be used by both `hass-tabs-subpage-data-table` and other places like here.

Reply:
That's probably going to be more difficult than I can tackle. Guess I'll drop this as I don't see a way forward.